src/Pos/Txp/Core/Types.hs

```diff
@@ -152,10 +152,16 @@ instance Buildable TxOut where
     build TxOut {..} =
         bprint ("TxOut "%coinF%" -> "%build) txOutValue txOutAddress
 
-type TxOutAux = (TxOut, [(StakeholderId, Coin)])
+-- | Transaction output and auxilary data corresponding to it.
+-- [CSL-366] Add more data.
```

Comment:
Why is there `TxOutAux` in `binary-protocols`? It's an internal type which isn't sent over wire, only stored in DB, isn't it? I don't think that such types should be described in `binary-protocols`. I am hot sure that it's not sent over wire though.